data/actionsmap/yunohost.yml

```diff
@@ -302,6 +302,14 @@ domain:
             arguments:
                 domain:
                     help: Target domain
+                -t:
+                    full: --ttl
+                    help: Time To Live (TTL) in second before DNS servers update. Default is 3600 second (i.e. 1 hour).
+                    extra:
+                        pattern: &pattern_ttl
```

Comment:
Note that `&pattern_ttl` allows to use that block in another part of the actions map - i.e. [this](https://github.com/YunoHost/yunohost/blob/unstable/data/actionsmap/yunohost.yml#L111) allows [that](https://github.com/YunoHost/yunohost/blob/unstable/data/actionsmap/yunohost.yml#L176).